tests/ci_build/run-build.sh

```diff
@@ -92,4 +94,11 @@ cmake -B "${build_dir}" \
 	"${CMAKE_SAUNAFS_ARGUMENTS[@]}" \
 	"${EXTRA_ARGUMENTS[@]}" "${WORKSPACE}"
 
-nice make -C "${build_dir}" -j "$(nproc)" install
+nice make -C "${build_dir}" -j "$(nproc)"
+if [ "${do_install}" = true ]; then
+	nice make -C "${build_dir}" install
+fi
+
+if [ -f "${build_dir}/CPackConfig.cmake" ]; then
```

Comment:
IMHO, it's a good idea to print a message notifying the user whether `CPackConfig.cmake` file is missing.